### ISSUE TITLE
`Sourcery: Create a default Sourcery configuration file for your project` command fails

### ISSUE THREAD

**Issue by willfrey** [sourcery-ai/sourcery-vscode#89]
Running the `Sourcery: Create a default Sourcery configuration file for your project` command on VS Code fails with the following error:

```
'ascii' codec can't encode character '\U0001fa84' in position 2: ordinal not in range(128)
```

It looks like it's trying to add 🪄 somewhere?

#### Relevant version info:
- Sourcery v0.12.6 
- VS Code v1.69.2
- macOS v12.5

**Comment by reka** [comment]
Hi @willfrey ,

Thanks a lot for reporting this :bug: .

> It looks like it's trying to add :magic_wand:  somewhere?

Yes, exactly. :+1: 

This command wants to:

* create a `.sourcery.yaml` file
* add some initial content to it

This initial content is starting with the line:

```
# 🪄 This is your project's Sourcery configuration file.
```

And unfortunately, this doesn't work with ascii.
We'll fix this and let you know when it's available.

**Comment by Hellebore** [comment]
Fixed with 0.12.9
